### DISCUSSION TITLE
Error in running the some samples in codesnadbox

### DISCUSSION THREAD

**Discussion by YJDoc2** [excaliburjs/Excalibur#2176] [Q&A] [answered]
While trying to run the demos shown on the excalibur website, some demos do not run in the codesandbox.
Particularly : The platformer and the breakout examples seem to be broken, while the other two are running quite well.

I'm running Firefox, v 95.0.1

In case of the platformer, the "mini-browser" in the codesandbox shows 502 bad gateway error, while for the breakout nothing is loaded/shown in the "mini-browser".

I have tried refreshing, trying to run it with random interval in between to see if there was any issue particularly at the time I was trying to access, but for all tries, both of the examples don't run.

Can someone confirm if this is reproducible, or just my issue?
Thanks!

**Comment by eonarheim** [comment, marked as answer]
The platformer definitely is no longer working for me in codesandbox, however the breakout did build for me and display for me in codesandbox. I pulled both locally as well to double check them. (platformer needed a small webpack tweak I've pushed, but breakout seems okay locally out of the box) 

I've had mixed luck with codesandbox recently, might be time to host on gh pages instead? One nice feature of codesandbox is the ability to easily play with the code samples and make changes.

@kamranayub @jedeen  Do you have any leads on a better tool? 

I'm half tempted to try and do what webglfundamentals does with [their docs](https://webglfundamentals.org/webgl/lessons/webgl-3d-textures.html) and build a custom embed. The tricky bit will be typescript/build tool support...

<img width="1124" alt="image" src="https://user-images.githubusercontent.com/612071/147578491-acd98d33-69ce-48be-a0d1-5bdbbaeecd96.png">

**Reply by eonarheim** [reply]
Good news! I was able to tweak the codesandbox config to get the sample going again

<img width="480" alt="image" src="https://user-images.githubusercontent.com/612071/147587081-6028b45f-3b0e-44f8-86e7-5030f77341b6.png">

**Reply by kamranayub** [reply]
May be good to pin all the dependency versions too so they don't accidentally drift over time. Then we could add some reminders to update them once a release or couple releases.

**Reply by YJDoc2** [reply]
Hey, Thanks for such a quick response.
Now the platformer works for me as well. The breakout also started working, but it took a lot of time to load and start the game, I only saw it working because I had kept it running in a background tab and forgotten about it :) So maybe its just some issue from my side.
